data/changelog.txt

```diff
@@ -138,12 +184,31 @@ Vehicles: make shock absorbers actually absorb shock damage.
 Fix NPC dying immediately on accepting inhaler mission
 Fix crash when attempting to butcher a corpse.
 Fix gear getting damaged when weather has no damaging effects.
+Improve invalidation of max_populated_zlev cache, fixes lighting in some teleportation scenarios.
+Prevent dodging or blocking while insensible.
+Fix apparent light asymmetry when sun shines through a single-tile hole in the roof.
+Made installation time mandatory for gunmods.
+Fix #44421 tents and vehicles are not blocking sunlight.
+Fix scent map saved with invalid utf8 sequence.
+Prevent monsters from bashing obstacles based on tracking sounds.
```

Comment:
This is a bit unclear. "based on" is meaning "only due to", or what?

Reply:
Only due to, i.e. they won't bash things unless they see or smell something, I agree it needs clarification, it's a pretty big change.
```suggestion
Monsters no longer bash obstacles based on sound. This prevents groups of monsters from getting stuck in a loop of bashing based on the sound of other monsters bashing.
```